### PR TITLE
Wios station data interface

### PR DESCRIPTION
**Pull request by MattBlack85**

### CODE REVIEW

File: tests/test_station_data_interface.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+def test_filter_bad_key(wios_station_data):
+    interface = StationDataInterface(wios_station_data)
+    assert interface._filter_series('foo') == []
+
+
+@pytest.mark.parametrize('attr', ['pm10_data', 'pm25_data'])
+def test_pm_data(attr, wios_station_data):
+    interface = StationDataInterface(wios_station_data)
+    # for some reasons there are 26 measurements for one single day
```
Comment (jszafran) [inline tests/test_station_data_interface.py:40]: Maybe this is due to fact that we're using CET (UTC+1h) and CEST (UTC+2h) in Poland?

File: krakus/utils/wios.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        except StopIteration:
+            return []
+
+    @property
+    def pm10_data(self) -> list:
+        return self._filter_series('pm10')
+
+    @property
+    def pm25_data(self) -> list:
+        return self._filter_series('pm2.5')
```
Comment (jszafran) [inline krakus/utils/wios.py:58]: Two things came to my mind:
1. How about exposing date as a property? We could either use startTime/extStartTime from Wios API response or do some regex on the title (I'm assuming title is always there). 
2. Another thing that could be useful for analysis would be having station name as property. I don't see it in API response though - so could be extracted from title as well?

File: krakus/utils/wios.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    def title(self) -> str:
+        return self.data['title']
+
+    @property
+    def series(self) -> list:
+        return self.data['series']
+
+    def _filter_series(self, key: str) -> list:
+        try:
+            raw = next(filter(lambda x: x['paramId'] == key, self.series))
```
Comment (MattBlack85) [inline krakus/utils/wios.py]: should pick a more meaningful name

**Review by jszafran — APPROVED** [review]
Looks good.

File: krakus/utils/wios.py (hunk 2 above)
Comment (MattBlack85) [reply krakus/utils/wios.py:58]: 1) do you mean the date as 2019-09-01 for example?
2) yes, that's a good point :ok_hand:, I will add it

Comment (jszafran) [reply krakus/utils/wios.py:58]: 1. Yep, having it as datetime.date or even string alone would enable convenient filtering of dataset.

Comment (MattBlack85) [reply krakus/utils/wios.py:58]: sure, np, will add it

Comment (jszafran) [reply krakus/utils/wios.py:58]: Also, having date & station_name as properties, some nice __repr__ could be implemented, maybe somethig like
`f'{self.__class__.__name__}({self.station}, {self.date})'`?

File: tests/test_station_data_interface.py (hunk 1 above)
Comment (MattBlack85) [reply tests/test_station_data_interface.py:40]: probably

File: krakus/utils/wios.py (hunk 2 above)
Comment (MattBlack85) [reply krakus/utils/wios.py:58]: @jszafran 1) and 2) have been targeted, for the repr --> https://github.com/MattBlack85/krakus/issues/21

please re-review

**Review by jszafran — APPROVED** [review]
Approved.

**Pull request merged**